### PR TITLE
Feature/account page

### PR DESCRIPTION
**Pull request by YevheniiBieliaiev**

### CODE REVIEW

**Comment by YevheniiBieliaiev** [comment]
image

**Comment by YevheniiBieliaiev** [comment]
image

File: packages/frontend/pages/user-account/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import type React from 'react';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import { wrapper } from 'store';
+import { AccountLayout } from '../../components/user-account-layout/layout';
+
+export const getServerSideProps = wrapper.getServerSideProps(
+  (store) => async (ctx) => {
+    const { locale } = ctx;
+    console.log(store);
```
Comment (Vitalii4as) [inline packages/frontend/pages/user-account/index.tsx:9]: remove

**Review by Vitalii4as — CHANGES_REQUESTED** [review]

File: packages/frontend/pages/user-account/index.tsx (hunk 1 above)
Comment (YevheniiBieliaiev) [reply packages/frontend/pages/user-account/index.tsx]: done

**Review by Vitalii4as — APPROVED** [review]

**Pull request merged**